### PR TITLE
Make Amount in DroppingAmmoEventArgs ushort instead of int

### PR DESCRIPTION
**Pull request by ghost**
DroppingAmmoEventArgs.Amount : int -> ushort

### CODE REVIEW

**Review by NaoUnderscore — APPROVED** [review]

File: Exiled.Events/EventArgs/Player/DroppingAmmoEventArgs.cs [hunk 1]
```diff
@@ -52,7 +52,7 @@ public DroppingAmmoEventArgs(Player player, AmmoType ammoType, ushort amount, bo
         /// <summary>
         ///     Gets or sets the amount of ammo being dropped.
         /// </summary>
-        public int Amount { get; set; }
```
Comment (louis1706) [inline Exiled.Events/EventArgs/Player/DroppingAmmoEventArgs.cs:55]: Use Obsolete instead of removing it and cast it to Ushort ?

**Review by louis1706 — CHANGES_REQUESTED** [review]

File: Exiled.Events/EventArgs/Player/DroppingAmmoEventArgs.cs (hunk 1 above)
Comment (NaoUnderscore) [reply Exiled.Events/EventArgs/Player/DroppingAmmoEventArgs.cs:55]: `ushort` is what the game uses, how is he going to cast it?

**Review by louis1706 — APPROVED** [review]
Nao said me than it's isn't a break change for Plugin (if it is i will blamed NAO for it)

**Comment by Bonjemus** [comment]
This pull is breaking bad.

**Pull request merged**